### DISCUSSION TITLE
Resolution on AppleTV 4K

### DISCUSSION THREAD

**Discussion by cryptedjeffrey** [NoahFetz/F1AppleTV#62] [General]
We are all aware that currently the F1TV Pro feeds are 1080p (1080/50 main feed).

My question is, if you run 4 feeds on an AppleTV 4K, will they all be 1080p? Totalling in 4K for the total screen?

Cheers

**Comment by NoahFetz** [comment]
Yes that should be the case in theory but I haven't counted the pixels yet ^^
